src/enums.ts

```diff
@@ -1,3 +1,36 @@
+export enum ContributingDevelopersVariableNames {
```

Comment:
these are environment vars right? - seems like they should be in constants rather than enums - then we can override too

Reply:
if we don't allow overrides and there is only a one-to-one mapping - make a "dictionary" of IntegrationName to the value string